lib/model.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const Objection = require('objection');
+const { ModelValidation } = Objection.ValidationError.Type;
```

Comment:
@devinivy yo! took a quick peek, the Type method isn't documented publicly (apparently just dug out of source:  https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/ValidationError.js#L11-L13 ), but it's values are documented e.g. http://vincit.github.io/objection.js/#error-handling

Does this feel ok to keep, maybe linking to where it came from? Or better to find some alternative?

Reply:
Hey @koskimas, is this a valid way to reference the `ModelValidation` type in a custom validator, supporting objection v1?